### DISCUSSION TITLE
Getting error when trying to use Speech to Text on CUDA container

### DISCUSSION THREAD

**Discussion by nikito** [open-webui/open-webui#6231] [General]
# Bug Report

## Installation Method

Pod in k8s

## Environment

- **Open WebUI Version:** 0.3.32
- **Ollama (if applicable):** 0.3.13 (running on separate pod)

- **Operating System:** k8s in Ubuntu 22.04
- **Browser (if applicable):** Chrome

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [x ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Speech to Text should successfully process the speech.

## Actual Behavior:

Speech to text fails, throwing error in backend, causing app to crash and restart.

## Description

**Bug Summary:**

When using the CUDA enabled container, upon trying to perform speech to text, the app crashes, and I see this error in my logs:
Could not load library libcudnn_ops_infer.so.8. Error: libcudnn_ops_infer.so.8: cannot open shared object file: No such file or directory

Looking in the container, it appears all my libraries are the .9 version, not .8. This is likely due to my cluster using a higher version of CUDA. Is it possible to make the app compatible with the later versions of CUDA as well?

## Reproduction Details

**Steps to Reproduce:**
Install the cuda variant of the container, with latest CUDA enabled container.

## Logs and Screenshots

**Browser Console Logs:**

**Docker Container Logs:**
<html>
<body>

2024-10-16 08:19:39.141 | Could not load library libcudnn_ops_infer.so.8. Error: libcudnn_ops_infer.so.8: cannot open shared object file: No such file or directory |  
-- | -- | --
  |   | 2024-10-16 08:19:38.594 | transcribe /app/backend/data/cache/audio/transcriptions/153c6178-0da2-4c9d-9687-68590fda567a.wav

</body>
</html>

**Comment by ptempier** [comment]
Having the same issue, it was also converted to a discussion .....
My understanding is that it a packaging issue of the cuda container image.
Because this file is a library file, not a driver file, hence it not loaded over by nvidia-continerd
